Bio/Phylo/NewickIO.py

```diff
@@ -292,7 +299,7 @@ def newickize(clade):
             if label:
                 unquoted_label = re.match(token_dict["unquoted node label"], label)
                 if (not unquoted_label) or (unquoted_label.end() < len(label)):
-                    label = "'%s'" % label.replace("\\", "\\\\").replace("'", "\\'")
+                    label = "'%s'" % label.replace("'", "''")
```

Comment:
I see nothing in the newick specification https://web.archive.org/web/20170714111939/https://evolution.genetics.washington.edu/phylip/newick_doc.html about slashes, but I wonder what else the original line of code here sought to handle.

Reply:
It looks like the backslash-escape was introduced by @bendmorris here: 83f824a618bf2623370ab0a53d3c88915e52a3e1

Prior to that, there was no special handling for backslashes or quotes within the clade name during serialization. It was originally based on the Nexus.Nexus serializer, then written for speed and NeXML compatibility.

But according to the Phylip spec above, quote-escaping should be done like Excel/CSV, not like Python syntax. I think it's safe to call this a bug and merge this PR.